controllers/profiles/unfollowUserV3.js

```diff
@@ -38,38 +40,86 @@ module.exports = async (req, res) => {
       return userFollowUser;
     });
   } catch (e) {
-    console.log('Error in unfollow user v2 sql transaction');
+    console.log('Error in unfollow user v3 sql transaction');
     return ErrorResponse.e409(res, e.message);
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's not a good practice to log errors without any context or additional information. It would be better to include the error message in the log output. This will help with debugging if an error occurs.
```diff
- console.log('Error in unfollow user v3 sql transaction');
+ console.log('Error in unfollow user v3 sql transaction', e.message);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->